bitbots_localization/config/config.yaml

```diff
@@ -12,17 +12,15 @@
   # ROS-Stuff #
   #############
 
-  line_topic: 'line_relative'
-  goal_topic: 'goals_simulated'
-  fieldboundary_topic: 'field_boundary_relative'
-  fieldboundary_in_image_topic: 'field_boundary_in_image'
-  corners_topic: 'corners'
-  tcrossings_topic: 'tcrossings'
-  crosses_topic: 'crosses'
+  line_topic: '/line_relative'
+  goal_topic: '/goals_simulated'
+  fieldboundary_topic: '/field_boundary_relative'
+  fieldboundary_in_image_topic: '/field_boundary_in_image'
+  corners_topic: '/corners'
+  tcrossings_topic: '/tcrossings'
+  crosses_topic: '/crosses'
```

Comment:
do we actually want this? this prevents us from using a namespace as far as I know. that could result in problems with simulation for example. Does it benefit us in any way?

Reply:
We currently use a namespace to publish our data. But This parameters define the subscribers, which are topics in the top level namesace or in the namespace of other nodes. Therefore this is the only working way. Without the slash we only sub to the topic `/bitbots_localization/field_boundary_in_image` etc. which is not the intended behavior